src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -358,7 +347,6 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
             stagingTasks.size()));
     
     ClusterState clusterState = ClusterState.getInstance();
-    DfsTask dfsTask = clusterState.getDfsTask(status.getTaskId());
 
     if (status.getState().equals(TaskState.TASK_FAILED)
         || status.getState().equals(TaskState.TASK_FINISHED)
```

Comment:
I would like to add this, but the build will not pass with the current version of mesos it compiles with. Will have to wait.